app/styles/components/category-item.scss

```diff
@@ -13,7 +13,7 @@
     font-size: $body-font-size-small;
     margin: 10px 0;
 
-    &.selected {
+    .selected & {
       color: $default-color;
```

Comment:
Whoops can you add an issue to fix that? I also realized that my overall BEM change is not really the direction I wanted to head in but was unwilling to spend more time on something not super critical.